composer/trainer/trainer_hparams.py

```diff
@@ -538,7 +546,20 @@ def validate(self):
 
         if (isinstance(self.grad_accum, str) and self.grad_accum != "auto") or (isinstance(self.grad_accum, int) and
                                                                                 self.grad_accum < 1):
-            raise ValueError('grad_accum must be "auto" or an int greater than or equal to 1')
+            raise ValueError('grad_accum must be "auto" or an int greater than or equal to 1.')
+
+        if self.autoresume:
```

Comment:
Don't worry about adding the validation in here -- since the hparams constructs the trainer, the same errors would be raised anyways. Unrelated, but I kinda want to remove all the hparams validate functions -- https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/